integrations/helm/operator/operator.go

```diff
@@ -343,3 +341,31 @@ func (c *Controller) deleteRelease(fhr ifv1.FluxHelmRelease) {
 	}
 	return
 }
+
+// needsUpdate compares two FluxHelmRelease and determines if any changes occurred
+func needsUpdate(old, new ifv1.FluxHelmRelease) bool {
```

Comment:
This is a bit brittle, but it'll do for now, unless you can think of a way of making sure it gets updated when the definition is updated.